apis/v1alpha1/httproute_types.go

```diff
@@ -465,22 +473,33 @@ type HTTPRequestMirrorFilter struct {
 
 	// BackendRef is a local object reference to mirror matched requests to. If
 	// both BackendRef and ServiceName are specified, ServiceName will be given
-	// precedence. If the referent cannot be found, controllers must set the
-	// "InvalidRoutes" status condition on any Gateway that includes this Route
-	// to true.
+	// precedence.
+	//
+	// If the referent cannot be found, the route must be dropped
+	// from the Gateway. The controller should raise the "ResolvedRefs"
```

Comment:
Especially in the case of mirroring, it seems appropriate to forego mirroring but leave the rest of the functionality intact.